cartography/intel/crxcavator.py

```diff
@@ -0,0 +1,210 @@
+import logging
+import json
+import requests
+import requests.auth
+import os
+
+from cartography.util import run_cleanup_job
+
+logger = logging.getLogger(__name__)
+
```

Comment:
Are there other assets under crxcavator that we might want to ingest in the future?  If so, I recommend making a cartography/intel/crxcavator folder and moving this logic to `crxcavator/__init__.py`.

Reply:
Currently crxcavator is limited to chrome extensions, but refactored to this model in case of future changes.